src/components/Collapsible/Collapsible.tsx

```diff
@@ -146,7 +175,7 @@ const Collapsible = createClass({
 						rootType,
 						{
 							...omitProps(passThroughs, Collapsible),
-							ref: this.storeRef('root'),
+							ref: (ref: any) => (this.rootRef = ref),
```

Comment:
can we avoid using `any` here? looks like in other components like Button we didn't need to specify the type

Reply:
This looks like solid info on what `ref`s actually are: https://medium.com/@martin_hotell/react-refs-with-typescript-a32d56c4d315